javascript/src/seedsource/containers/Map.jsx

```diff
@@ -127,45 +131,54 @@ class Map extends React.Component {
                     subdomains: ['server', 'services']
                 }),
                 L.tileLayer('//{s}.arcgisonline.com/ArcGIS/rest/services/World_Terrain_Base/MapServer/tile/{z}/{y}/{x}', {
-	                attribution: 'Tiles &copy; Esri &mdash; Source: USGS, Esri, TANA, DeLorme, and NPS',
-	                maxZoom: 13,
+                    attribution: 'Tiles &copy; Esri &mdash; Source: USGS, Esri, TANA, DeLorme, and NPS',
+                    maxZoom: 13,
                     subdomains: ['server', 'services']
                 }),
                 L.tileLayer(
                     '//{s}.arcgisonline.com/ArcGIS/rest/services/Canvas/World_Light_Gray_Base/MapServer/tile/{z}/{y}/{x}', {
-                    attribution: 'Tiles &copy; Esri &mdash; Esri, DeLorme, NAVTEQ',
-                    maxZoom: 16,
-                    subdomains: ['server', 'services']
-                }),
+                        attribution: 'Tiles &copy; Esri &mdash; Esri, DeLorme, NAVTEQ',
```

Comment:
These last two `tileLayer` blocks are indented too much (it's hard to see here, but look at it in an editor).